src/features/header-flyout/styles/HeaderFlyout.scss

```diff
@@ -69,3 +66,56 @@
         }
     }
 }
+
+@include breakpoint($medium-screen) {
```

Comment:
Do we need to check for `isResponsive` similar to OverlayHeader?

Reply:
I think this can be handled in the parent project, but I should probably add it in anyways